tel-search-connector/processes/PhoneDirectory.mod

```diff
@@ -36,6 +36,7 @@ Py0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 </elementInfo>
 ' #txt
 Py0 f0 81 49 30 30 -13 17 #rect
+Py0 f0 res:/webContent/icons/searchCH200x200.png?small #fDecoratorIcon

```

Comment:
you may also set this icon on config/rest-clients.yaml ... there's an icon picker next to the name